Components/BlockMediaText/howto.md

```diff
@@ -26,3 +26,16 @@ add_filter('Flynt/addComponentData?name=BlockMediaText', function ($data) {
     }
 });
 ```
+
+**Beware:** If you use this component as a static component (or any component with tabs), you need to add a closing tab*, otherwise the next components will show up inside the last tab.
+
+*A tab with an empty label, and `"endpoint": 1`
+
+```json
+{
+  "name": "endTab",
+  "label": "",
+  "type": "tab",
+  "endpoint": 1
+}
+```
```

Comment:
Tried already, it doesn't work neither. It breaks the tabs in "visually 2 components" and my next wysiwyg shoes up in the tab settings.
Tried already multiple things before ending with this solution.